hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/file/txns/FileUpdateResourceUsage.java

```diff
@@ -64,6 +65,24 @@ public FeeData usageGiven(final TransactionBody txn, final SigValueObj svo, fina
         }
     }
 
+    public FeeData usageGiven(final TransactionBody txn, final SigValueObj svo, final File file) {
```

Comment:
Was this just missing? Trying to figure out where this came from

Reply:
I created similar method that instead using State will use specific file and it will be deprecated when we get rid of mono